.changes/unreleased/BUG FIXES-20230803-120411.yaml

```
@@ -0,0 +1,6 @@
+kind: BUG FIXES
+body: 'types/basetypes: Fix Float64Attribute bug by preserving precision from number
+  values'
```

Nit: Maybe this is clearer for developers?

```suggestion
body: 'types/basetypes: Prevented Float64Value Terraform data consistency errors for
  numbers with high precision floating point rounding errors'
```

---

Sounds good to me 👍🏻